config/routes.rb

```diff
@@ -128,8 +125,8 @@
     end
 
     # Confirmation
-    get    '/users/confirmation/new' => 'users/confirmations#new', :as => :new_user_confirmation
-    get    '/users/confirmation' => 'users/confirmations#show', :as => :user_confirmation
+    get    '/users/confirmation/new' => 'users/confirmations#new', as: :new_user_confirmation
+    get    '/users/confirmation' => 'users/confirmations#show', as: :user_confirmation
```

Comment:
Line is too long. [86/80]<br>Unnecessary spacing detected.<br>Put one space between the method name and the first argument.<br>Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.